pkg/cloud/services/eks/nodegroup.go

```diff
@@ -38,8 +38,14 @@ import (
 	"sigs.k8s.io/cluster-api-provider-aws/v2/pkg/record"
 	clusterv1 "sigs.k8s.io/cluster-api/api/v1beta1"
 	"sigs.k8s.io/cluster-api/util/annotations"
+	"sigs.k8s.io/cluster-api/util/conditions"
 )
 
+var IgnoreLifecycleHooks = map[string]bool{
```

Comment:
Please add a short comment where these come from (I guess precreated by EKS, but not documented?)